src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://bootswatch.com/4/yeti/bootstrap.min.css">
+  <title>LeaderBoard</title>
+</head>
+<body>
+  
+  <div class="container mt-3">
+    <div class=" h1 font-weight-bold">Leaderboard</div>
+    <div class="row"> 
+    <div class="inner-container d-flex col-md-6">
+      <div class="scores-container">
+        <div class="d-flex">
+          <div class="h4 font-weight-bold">Recent scores</div>
+          <button type="button" class="btn btn-primary btn-sm"> Refresh</button>
+        </div>
+        <div class="">
+          <table class="table table-striped mt-4">
+            <thead class="thead-dark">
+                <tr>
+                  <th class="display-5 lead text-white">Name</th>
+                  <th class="display-5 lead text-white">Scores</th>                 
+                </tr>
+            </thead>
+            <tbody id="boardList">
+            </tbody>
+          </table>
+        </div>
+      </div>
+      <div class="form-container">
+        <div class="h4 font-weight-bold">Add your score</div>
+        <form action="#" class="d-flex form">
+          <input type="text" class="name form-control mb-4" placeholder="Enter name">
+          <input type="number" class="score form-control mb-4" placeholder="Enter score">
+          <input type="submit" value="submit" id="submit" class="btn btn-success btn-sm">
+        </form>
+      </div>
+    </div>
+    </div>
+  </div>
```

Comment:
- [ ] I kindly invite you to build all this in the `index.js` file. As a good practice, the index.html main contain only the root element (index.js in this case). 🐱
- Don't forget to use all ES6 best practices for this.